akka-docs/rst/scala/http/routing-dsl/directives/route-directives/redirect.rst

```diff
@@ -15,7 +15,7 @@ Description
 Completes the request with a redirection response to a given targer URI and of a given redirection type (status code).
 
 ``redirect`` is a convenience helper for completing the request with a redirection response.
-It is equivalent to this snippet relying on the ``complete`` directive:
+It is equivalent to this snippet relying on the ``complete`` method on ``RequestContext`` (a directive is also available):
 
 .. includecode2:: /../../akka-http/src/main/scala/akka/http/scaladsl/server/directives/RouteDirectives.scala
```

Comment:
that include breaks now, since you moved the code, but didn't update the include